app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express")
+const getTopics  = require("./controllers/get-topics.controller")
+const app = express()
+
+app.get("/api/topics", getTopics)
+
+app.use((error, request, response, next) => {
+    // General error, will be updated as needed
+    return response.status(400).send(error)
```

Comment:
Is a 400 status code appropriate for a general error? Is there another status code you can think of that might be more appropriate at this stage?

Reply:
I updated it with 500.